presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/MetastoreUtil.java

```diff
@@ -135,23 +135,23 @@ private static Properties getHiveSchema(
 
         schema.setProperty(SERIALIZATION_DDL, toThriftDdl(tableName, dataColumns));
 
-        String partString = "";
+        StringBuilder partString = new StringBuilder();
         String partStringSep = "";
-        String partTypesString = "";
+        StringBuilder partTypesString = new StringBuilder();
         String partTypesStringSep = "";
         for (Column partKey : partitionKeys) {
-            partString += partStringSep;
-            partString += partKey.getName();
-            partTypesString += partTypesStringSep;
-            partTypesString += partKey.getType().getHiveTypeName().toString();
+            partString.append(partStringSep);
+            partString.append(partKey.getName());
+            partTypesString.append(partTypesStringSep);
+            partTypesString.append(partKey.getType().getHiveTypeName().toString());
             if (partStringSep.length() == 0) {
                 partStringSep = "/";
                 partTypesStringSep = ":";
```

Comment:
StringBuilder is good, but List.Builder<String> + `String.join("/", list)` would be even better